speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/SQLDatabaseStorage.cpp

```diff
@@ -100,13 +141,19 @@ bool SQLDatabaseStorage::OpenDatabase(
     connection_properties["CLIENT_MULTI_STATEMENTS"] = false;
     connection_properties["sslEnforce"] = true;
 
+    if (p_useEncryption == "true")
```

Comment:
Might want to add to_lower to ensure putting "True" or "TRUE" or similar spellings work as well